zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/JTensor.java

```diff
@@ -137,4 +162,52 @@ public String stringValue() {
     Preconditions.checkState(dataType() == DataType.DOUBLE);
     return (double []) data();
   }
+
+  private static int[] toIntExact(final long[] dimensions) {
+    // stream would look nice but this is probably faster
+    final int[] intDimensions = new int[dimensions.length];
+    for (int i = 0; i < dimensions.length; i++) {
+      intDimensions[i] = Math.toIntExact(dimensions[i]);
+    }
+    return intDimensions;
+  }
+
+  @VisibleForTesting
+  static Object toStringArray(
```

Comment:
I would either make it private or move it to a separate class (i.e `ArrayUtils` / `ArrayExtras`). But, I think it would be ok to make it private and then just evaluate the `value()` of the `Tensor` as you do in `assertArrayEquals(expectedValue, jt.value());`. I just don't like having it exposed internally through `Jtensor` api.